src/applications/edu-benefits/1990/pages/benefitsEligibilityUpdate.js

```diff
@@ -0,0 +1,102 @@
+import { isProductionOfTestProdEnv } from '../helpers';
+import { benefitsLabels } from '../../utils/labels';
+
+export const benefitsEligibilityUpdate = (
+  benefitsEligibilityBox,
+  validateBooleanGroup,
+  chapter33,
+  chapter30,
+  chapter1606,
+) => {
+  const uiSchema = isProductionOfTestProdEnv()
+    ? {
+        'ui:description': benefitsEligibilityBox,
+        'view:selectedBenefits': {
+          'ui:title': 'Select the benefit that is the best match for you.',
+          'ui:validations': [validateBooleanGroup],
+          'ui:errorMessages': {
+            atLeastOne: 'Please select at least one benefit',
+          },
+          'ui:options': {
+            showFieldLabel: true,
+          },
+          chapter33: {
+            'ui:title': benefitsLabels.chapter33,
+            'ui:options': {
+              expandUnderClassNames: 'schemaform-expandUnder-indent',
+            },
+          },
+          chapter30: {
+            'ui:title': benefitsLabels.chapter30,
+          },
+          chapter1606: {
+            'ui:title': benefitsLabels.chapter1606,
+          },
+        },
+      }
+    : {
+        'ui:description': benefitsEligibilityBox,
+        'view:selectedBenefits': {
+          'ui:title': 'Select the benefit that is the best match for you.',
+          'ui:errorMessages': {
+            atLeastOne: 'Please select at least one benefit',
+          },
+          'ui:options': {
+            showFieldLabel: true,
+          },
+          chapter33: {
+            'ui:widget': 'radio',
+            'ui:required': formData => !formData.chapter33,
+            'ui:options': {
+              labels: benefitsLabels,
+              hideLabelText: true,
+            },
+            'ui:errorMessages': {
+              required: 'Please select at least one benefit',
```

Comment:
Now that we are making this a radio button that will only allow 1 selection, the error message should be updated. My recommendation is: required: "You must select a benefit"